.gitignore

```diff
@@ -1,8 +1,8 @@
-# :::::::::::::::::::::: cut ::::::::::::::::::::::
-
-# Put ignore patterns for build artifacts above the cut line
-# so they are ignored in the source repo and committed
-# to Pantheon.
+# :::::::::::::::::::::: cut ::::::::::::::::::::::
```

Comment:
Just curious, what is this for?

Reply:
I'd just left it in from it existing from other code. It's not there for me.